SPECS/filesystem/filesystem.spec

```diff
@@ -1,7 +1,7 @@
 Summary:      Default file system
 Name:         filesystem
 Version:      1.1
-Release:      17%{?dist}
+Release:      18%{?dist}
```

Comment:
File system is a `toolchain` package; We missed to bump the relevant entries in toolchain `manifest` files.